dask/dataframe/core.py

```diff
@@ -736,14 +736,14 @@ def map_partitions(self, func, *args, **kwargs):
         Parameters
         ----------
         func : function
-            Function applied to each partition.
+            Map function applied to each partition. If this function accepts the special ``partition_info`` keyword
```

Comment:
```suggestion
            The function applied to each partition. If this function accepts the special ``partition_info`` keyword
```

I find calling this a "Map function" a bit confusing, since it's the function being mapped. I think dropping the `Map` makes sense here, but no strong thoughts.